src/main/kotlin/com/terraformation/backend/tracking/model/ObservationResultsModel.kt

```diff
@@ -423,3 +453,18 @@ fun List<ObservationSpeciesResultsModel>.unionSpecies(
         )
       }
 }
+
+/** Calculate standard deviation */
+fun Collection<Int>.calculateStandardDeviation(): Int? {
+  if (this.size <= 1) {
+    return null
+  }
+
+  val numSamples = this.size.toDouble()
+  val samples = this.map { it.toDouble() }
+  val mean = samples.sumOf { it } / numSamples
+  val sumSquaredDifferences = samples.sumOf { (it - mean) * (it - mean) }
+  val variance = sumSquaredDifferences / (numSamples - 1)
+
+  return sqrt(variance).roundToInt()
+}
```

Comment:
I'm curious why we want to represent standard deviations as integers. Are we expecting the numbers to be large enough that the fractional parts won't be relevant?

Reply:
My rationale is that the unit of Std Dev should be the same as the samples. I should check back with product on this.

Reply:
Confirmed that standard deviation should use integer precision